recommendations/linkerd.md

```diff
@@ -0,0 +1,185 @@
+  [STEERING.md](https://github.com/linkerd/linkerd2/blob/main/STEERING.md)
+- Maintainers and Steering Committee members:
+  [MAINTAINERS.md](https://github.com/linkerd/linkerd2/blob/main/MAINTAINERS.md)
+
+#### Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website). For a specification, have a list of adopters for the implementation(s) of the spec.
+
+Yes, [ADOPTERS.md](https://github.com/linkerd/linkerd2/blob/main/ADOPTERS.md).
+
+## CNCF IP Policy
+
+External dependencies (including licenses):
+
+As of 3/25/2021, see the report here:
+
+https://gist.github.com/olix0r/38759123e08dfe5b35c4b1dd6cc52da5
+
+## Other Considerations
+
+### Alignment with Cloud Native
+
+_Does the project meet the definition of Cloud Native? The CNCF charter states:_
+
+_“Cloud native technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach._
+
+_“These techniques enable loosely coupled systems that are resilient, manageable, and observable. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil.”_
+
+Yes. Linkerd is a service mesh, one of the examples of cloud native
+technologies in the definition above.
+
+### Project and Code Quality
+
+#### Are there any metrics around code quality? Are there good examples of code reviews? Are there enforced coding standards?
+
+The Linkerd project maintains a strong history of code review, which is where
+quality and standards are enforced. Some recent examples include:
+
+https://github.com/linkerd/linkerd2/pull/5921
+https://github.com/linkerd/linkerd2/pull/5829
+https://github.com/linkerd/linkerd2/pull/5814
+https://github.com/linkerd/linkerd2-proxy/pull/954
+https://github.com/linkerd/linkerd2-proxy/pull/955
+
+#### What is the CI/CD system? Are there code coverage metrics? What types of tests exist?
+
+Linkerd uses a comprehensive suite of unit tests, integration tests, static
+analysis, powered by GitHub actions. Almost all tests run on GitHub-provided
+VMs, with minor exceptions (such as ARM build validation) that run on
+CNCF-provided Equinix servers. See https://github.com/linkerd/linkerd2/actions for details.
+
+A CNCF-sponsored fuzzing project with Ada Logics is underway.
+
+Releases are also created via GitHub Actions and the workflows are available
+in the same URL as above.
+
+#### Has a security assessment by the security SIG been done? If not, what is the status/progress of the assessment?
+
+Per the proposal, Linkerd completed a CNCF-sponsored [Cure53 security
+audit](https://github.com/linkerd/linkerd2/blob/main/SECURITY_AUDIT.pdf) in
+June 2019.
+
+## SIG Network recommendation
+
+**Open Questions**
+
+1. Published project scope - https://github.com/linkerd/rfc/issues/37
```

Comment:
Have a followup question for you in here. Would love to know more about what you're looking for.